src/api/video/video.controller.js

```diff
@@ -0,0 +1,39 @@
+const Video = require('./video.model');
+
+async function getVideos(req, res, next) {
+  try {
+    let response;
+    if ('id' in req.params) {
+      if (!/^[a-fA-F0-9]{24}$/.test(req.params.id)) throw new ReferenceError('Invalid Video ID.');
+      try {
+        response = [await Video.findById(req.params.id).exec()];
+        if (response[0] === null) throw new ReferenceError('The requested ID does not exist.');
+      } catch ({ message }) {
+        throw new ReferenceError(message);
+      }
+    } else response = await Video.find({}).exec();
+
+    const finalResponse = response.map(({ id, type, name, videoID, title, date, description, url, thumbnail, createdAt, updatedAt }) => ({
```

Comment:
The current prettier config allows this, perhaps we should set `prettierWidth` to something lower than 200. What do you think @spiray

Reply:
The default is 80 but I think that is too low. Should we do 160?

Reply:
It has to be at least 100/120 to affect this line. But setting it that low will also affect a lot of other files.

Reply:
I am personally fine with 200, I never had a problem with line lengths. If it becomes an issue for any contributors we can revisit this in the future.

Reply:
I agree, let's leave it for now.